README.md

```diff
@@ -20,13 +18,88 @@ sharp('input.jpg', {limitInputPixels:false})
     .catch(console.error);
 ```
 
-Many options are available. 
+Many options are available.
 
 [Check sharp documentation on TIFF output format.](https://sharp.pixelplumbing.com/api-output#tiff)
 
-
 ## Prerequisites for remote files - HTTP range requests
-
 This library works by loading only parts of the remote TIFF file (which may be huge). For this, the remote http server has to be compatible. Most production-ready http servers are compatible, but some development servers (such as `python3 -m http.server` or PHP built-in development web server) are not.
 
 [Check Mozilla documentation on HTTP range requests.](https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests)
+
+## Usage
+
+### Installation
+
+The plugin is available as both an npm package and a standalone script.
+
+#### Using NPM
+```bash
+npm i geotiff-tilesource
+```
+
+#### Using standalone scripts
+Standalone scripts for the plugin are available as both UMD and ES module scripts. The UMD script is compatible with the `OpenSeadragon` global object, while the ES module script can be imported as a module.
+
+```html
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/4.1.0/openseadragon.min.js"></script>
+
+<!-- Using UMD script -->
```

Comment:
Should we include the /dist folder with the compiled files into the repo? Or are they supposed to be generated by an automated build action and will be available after that happens?

Also, shouldn't importing as a module be possible just from the `/src/GeoTiffTileSource.js` file? Why is the `.es` prefix needed for the file type?

Also, we had discussed making the modules `.mjs` files - are you still up for that?